### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by ngeelani48**
**In this pull request following updates are added:**

- Implement the design from the sneak peek wireframes.
- Simple Styling but focused on making sure everything functions first. 
- Analyzed the wireframes deeply and try to.
- Identified all the necessary elements.
- Navigation made from one page to another.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,2 +1,44 @@
...
+      <div class="user-info">
+        <h3><%= @user.name %></h3>
+        <p class="posts-counter">Number of posts: <%= @user.posts_counter || 0 %></p>
+      </div>
+    </div>
+    <%# End of user info %>
+    <%# Posts start %>
+    <article>
+      <% @posts.each do |post| %>
+        <%= link_to user_post_path(@user, post) do %>
```
Comment (KDlamini) [inline app/views/users/show.html.erb]: kindly use the `#three_recent_posts` method that you declared in the user model to render the user's 3 most recent posts as required by the project requirements.

image

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,7 +1,37 @@
...
+          </div>
+          <div class="comment-like-container">
+            <p class="comment-likes-data">
+              Comment: <%= post.comments.count %>, Likes: <%= post.likes.count %>
+            </p>
+          </div>
+        </div>
+      <% end %>
+    <% end %>
+  </article>
```
Comment (KDlamini) [inline app/views/posts/index.html.erb]: Kindly use `post.recent_comments` (in your case `most_recent_comments`) to render 5 most recent comments in the `app/views/posts/index.html.erb` page.

For example:
```ruby
 <% post.recent_comments.each do |comment| %>
  <p><%= comment.author.name %>: <%= comment.text %></p> 
 <% end %>
```

- Users should see `5 recent comments` for each post.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @ngeelani48 🙋‍♀️,  

Great work so far! 💯 
You have done a great job creating views for the Blop app. 🎊 🥳 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞

✔️  Good PR and description
✔️  Users controller `#index` action renders all users to their corresponding Views index page
✔️  `See all Post` link opens the user's post index
✔️  Well documented and professional README

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (ngeelani48) [reply app/views/posts/index.html.erb]: @KDlamini Thanks for the review and I just forgot to include that. thanks for mentioning it.

File: app/views/users/show.html.erb (hunk 1 above)
Comment (ngeelani48) [reply app/views/users/show.html.erb]: @KDlamini Thanks for the review and I just forgot to include that. thanks for mentioning it.

**Review by victorokoroji — APPROVED** [review]
Hi @ngeelani48, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested changes implemented ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**